tests/pagination/tests.py

```diff
@@ -302,7 +302,7 @@ def test_page_getitem(self):
         # Make sure object_list queryset is not evaluated by an invalid __getitem__ call.
         # (this happens from the template engine when using eg: {% page_obj.has_previous %})
         self.assertIsNone(p.object_list._result_cache)
-        with self.assertRaises(TypeError):
+        with self.assertRaisesMessage(TypeError, ''):
```

Comment:
This is asserting '' is in the exception message which doesn't do anything useful.